hivemind/utils/grpc.py

```diff
@@ -10,6 +10,7 @@
 import grpc
 import numpy as np
 import torch
+import hivemind
```

Comment:
Better to use more specific imports, especially when only one function is used. Also, local imports need to be separated by a newline